gce_image_import_export_tests/test_suites/import/image_import_test_suite.go

```diff
@@ -194,5 +187,20 @@ func verifyImportedImageWithParams(ctx context.Context, testCase *junitxml.TestC
 	} else {
 		logger.Printf("Image '%v' cleaned up.", imageName)
 	}
+}
 
+func contains(arr []string, subarr []string) bool {
```

Comment:
This code is fine for this use case (array lookup where n<10), just make sure we don't use this for large amount of data as it's O(n^2) complexity